helm-frontend/src/components/LeaderboardTables.tsx

```diff
@@ -37,6 +39,63 @@ export default function LeaderboardTables({
   const [filteredModels, setFilteredModels] =
     useState<string[]>(modelsToFilter);
 
+  const [schema, setSchema] = useState<Schema | undefined>(undefined);
+
+  useEffect(() => {
+    const controller = new AbortController();
+    async function fetchData() {
+      const schema = await getSchema(controller.signal);
+      setSchema(schema);
+    }
+
+    void fetchData();
+    return () => controller.abort();
+  }, []);
+
+  const getModelForRunName = (model: string): string => {
+    if (schema) {
+      const foundItem = schema.models.find(
+        (item) => item.display_name === model,
+      );
+      if (foundItem) {
+        let toRet = foundItem.name;
+        if (toRet.includes("/")) {
+          toRet = toRet.replace("/", "_");
+          return toRet;
+        } else {
+          return toRet;
+        }
+      }
+    }
+    return "";
+  };
+
+  // create delimiter to parse out run group name (need to replace hyphen as some run names have hyphens in them)
+  function replaceLastHyphen(str: string): string {
+    const lastIndex = str.lastIndexOf(" - ");
+    if (lastIndex === -1) {
+      return str;
+    }
+    return str.substring(0, lastIndex) + "*" + str.substring(lastIndex + 1);
+  }
+
+  const getGroupForRunName = (rawGroup: string): string => {
+    const groupSplit = replaceLastHyphen(rawGroup).split("*");
+    const group = groupSplit[0].trim();
+    console.log(group);
+    console.log(schema?.run_groups);
```

Comment:
Delete this.

Reply:
I'll handle this and #2 before merging, then handle #1 in the subsequent PR.

Reply:
Handled this, but leaving out the second change as it changes the style in an ugly way (making every thing a bright blue link).